docker/vars.local.sh

```diff
@@ -5,4 +5,8 @@ export GP2GP_AMQP_BROKERS="amqp://activemq:5672"
 export GP2GP_MONGO_URI="mongodb://mongodb:27017"
 export GP2GP_MONGO_DATABASE_NAME="gp2gp"
 export GP2GP_MHS_OUTBOUND_URL="http://mock-mhs-adaptor:8081/mock-mhs-endpoint"
-export GP2GP_GPC_GET_URL="http://wiremock:8080/GP0001/STU3/1/gpconnect"
+export GP2GP_GPC_GET_URL="http://gpcc:8080/GP0001/STU3/1/gpconnect"
+
+export GPC_CONSUMER_URL="http://gpcc:8080"
```

Comment:
i think this ones okay to be localhost

Reply:
localhost will result in connections refused error. Host name needs to be the same as the container for the internal requests to work.